web/styles/modal.css

```diff
@@ -348,6 +348,7 @@
 
 .modal__spinner {
     display: flex;
+    align-items: center;
```

Comment:
If you're changing styles that affect all spinners, do that in a separate commit and explain why you think this shouldn't cause bugs with other spinners.